src/fetching/tasks/getTaskAssignments.ts

```diff
@@ -0,0 +1,13 @@
+import { defaultFetch } from '..';
+import { ZetkinTaskAssignment } from 'types/zetkin';
+
+const getTaskAssignments = (orgId : string, taskId: string, fetch = defaultFetch) => {
```

Comment:
These objects are called "Assigned tasks" in the API, not "Task assignments". I think this should be reflected here by renaming this (and other occurrences) to `getAssignedTasks`.